### ISSUE TITLE
Where's the description? 

### ISSUE THREAD

**Issue by DonRichards** [Polymer/web-component-tester#402]
No description. What is this repo?

**Comment by maxknee** [comment]
@DonRichards https://github.com/Polymer/web-component-tester/blob/master/README.md
Should help explain

**Comment by DonRichards** [comment]
yes but when searching repos most people aren't going into each and every repo and reading the opening section of the README to see it applies to their search. They use use the built in description. 

Right now this repos says "No description or website provided." But in a search is says nothing at all. My suggestion is to help make your repo more discoverable. 

![aaa screenshot](https://cloud.githubusercontent.com/assets/2738244/23129882/5c166b18-f752-11e6-8cd6-fb3b660d9647.png)
